### ISSUE TITLE
Tenant short_name vs long_name

### ISSUE THREAD

**Issue by ryscher** [CDL-Dryad/dryad-product-roadmap#2611]
Review the usage of short_name and long_name -- are both really needed? It seems like we use short_name for almost everything.

**Comment by sfisher** [comment]
I agree that we could probably dump the long name.  I think it's also a UC specific and a lot of universities only have two names and we don't need 3 versions of a name or there are better ways to do this (such as an unlimited number of aliases if we really need that functionality for a drop-down list or something else which I don't think we have stuff like that now except from ROR and doesn't need to be in our config).

UC crazy example: "UCB", "UC Berkeley", "University of California, Berkeley"  (or if you want to add another alias, "University of California" usually also means the Berkeley campus if it's not referring to the whole system since they claim that name because they were the first UC campus).

Stepping back a bit, I know one thing we use the short name for is for selecting from a list after first login for partner institutions.  It makes me wonder if we could just use ROR for an autocomplete here, also and just match up partners by their RORs for special benefits.  This would also have the advantage of collecting more institution information for users, earlier on in the process which might be nice for Dryad to understand where users are coming from in better granularity outside of the email addresses (which we don't get from ORCID a lot of times).
